backend/lambdas/api/authorizer/tests/test_get_user.py

```diff
@@ -120,48 +150,83 @@ def test_get_nonexistent_user(self):
         """
         MockUser.users = copy.deepcopy(USERS)
         email = "first.last@doesnotexist.com"
+        expected_userid = MockUser.users[-1].get("id") + 1
```

Comment:
Changed this to a derived value which still tests that a new user was created, but does not require the unit test to be hard-coded and updated when unrelated tests are added.